views/requests.erb

```diff
@@ -31,12 +32,14 @@
           <% end %>
         </td>
         <td>
+          <span aria-hidden="true">Requested:</span>
           <%= hold_request.request_date %>
         </td>
         <td>
+          <span aria-hidden="true">Expires:</span>
           <%= hold_request.request_date %>
```

Comment:
@niquerio `Line 36` and `Line 40` are pulling in the same method. Is this intentional?

Reply:
nope, not intentional. I added expiry.